no change     /shared/centos7/anaconda3/3.7/condabin/conda
no change     /shared/centos7/anaconda3/3.7/bin/conda
no change     /shared/centos7/anaconda3/3.7/bin/conda-env
no change     /shared/centos7/anaconda3/3.7/bin/activate
no change     /shared/centos7/anaconda3/3.7/bin/deactivate
no change     /shared/centos7/anaconda3/3.7/etc/profile.d/conda.sh
no change     /shared/centos7/anaconda3/3.7/etc/fish/conf.d/conda.fish
no change     /shared/centos7/anaconda3/3.7/shell/condabin/Conda.psm1
no change     /shared/centos7/anaconda3/3.7/shell/condabin/conda-hook.ps1
no change     /shared/centos7/anaconda3/3.7/lib/python3.7/site-packages/xontrib/conda.xsh
no change     /shared/centos7/anaconda3/3.7/etc/profile.d/conda.csh
no change     /home/agarwal.rishi/.bashrc
No action taken.
Traceback (most recent call last):
  File "Routine_Square_Code.py", line 588, in <module>
    main(sys.argv[1], int(sys.argv[2]))
IndexError: list index out of range
/var/spool/slurm/d/job23845441/slurm_script: line 13: [27.436407359332986,-7.57841301205225],: command not found
/var/spool/slurm/d/job23845441/slurm_script: line 14: [27.436407359332986,-7.518171474050515],: command not found
/var/spool/slurm/d/job23845441/slurm_script: line 15: [27.350233348102517,-7.518171474050515]]: command not found
